rst_files/mccall_model.rst

```diff
@@ -347,64 +319,36 @@ Our initial guess :math:`v` is the value of accepting at every given wage
 
 .. code:: julia
 
-    function plot_value_function_seq(plt,
-                                     c = 25,
-                                     β = 0.99,
-                                     num_plots = 6)
-
-        plt = deepcopy(plt)
-        v = collect(w_vals ./ (1 - β))
-        v_next = similar(v)
-
-        for i in 1:num_plots
-            plot!(plt, w_vals, v, label = "iterate $i")
-
-            # Update guess
-            for (j, w) in enumerate(w_vals)
-                stop_val = w / (1 - β)
-                cont_val = c + β * sum(v .* p_vals)
-                v_next[j] = max(stop_val, cont_val)
-            end
-            v[:] .= v_next
-        end
-        return plt
+    # parameters and constant objects
+    w_min, w_max = 10, 60
+    w_vals = range(w_min, w_max, length = n + 1)
+    c = 25
+    β = 0.99
+    num_plots = 6
+    E = expectation(dist)
+
+    # functions and matrix to plot
+    stop_val(w) = w / (1 - β)
```

Comment:
Having the `stop_val` and `cont_val` just seems to make the code further from the math.  Why not combine into the `best_val` function and get rid of them, or am I missing something?

Finally, instead of `best_val` why not make it closer to the math by calling it `T`?  That is what it does?

Reply:
You're right. Will implement these changes.